src/lib/rest/clientSocketHttp.cpp

```diff
@@ -291,8 +292,15 @@ std::string sendHttpSocket
   }
 
   // ----- Accept
-  headers = curl_slist_append(headers, "Accept: application/xml, application/json");
-  outgoingMsgSize += 41; // from "Accept: application/xml, application/json"
+  std::string acceptedFormats = "application/xml, application/json";
```

Comment:
Yes, if the forward is sent as XML, the Accept is set as XML also.
I added a FIXME about this. The broker could be more relaxed and accept both XML and JSON.